webcomponents/src/index.html

```diff
@@ -7,5 +7,15 @@
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <link rel="icon" type="image/x-icon" href="favicon.ico" />
   </head>
-  <body></body>
+  <body>
```

Comment:
Great, I had that in mind, if you look at my PR in the bash script I started to try to generate the list (in comment)... but was not very familiar with bash.. We can manage it manually.

Thanks